### PR TITLE
Feature automatic DL and update of seasonals

### PR DESCRIPTION
**Pull request by dymattic**
Added cli Flag and logic for triggering the seasonals download without asking for user input

### CODE REVIEW

File: anipy_cli/cli.py [hunk 1]
```diff
@@ -306,14 +312,16 @@ def list_possible(self, latest_urls):
 
     def download_latest(self):
         latest_urls = Seasonal().latest_eps()
+        pprint(latest_urls)
```
Comment (sdaqo) [inline anipy_cli/cli.py]: Lets remove that, its just confusing.

**Comment by sdaqo** [comment]
yeah that looks like a useful function, lets add it

**Comment by dymattic** [comment]
I will try to figure out a nice way to check whether an episode has been downloaded successfully or not.

Because right now it keeps re-downloading the last episodes in seasonals file ^^ @sdaqo

**Comment by sdaqo** [comment]
@dymattic  It does? It should actually do this already.

**Comment by dymattic** [comment]
> @dymattic It does? It should actually do this already.

Yea at least with the -f Flag it does.
I kinda always use the -f flag because the internal downloader seems buggy rn.

**Comment by dymattic** [comment]
@sdaqo OMG I know why this is happening... My "Bugfix" introduced this behaviour

**Comment by sdaqo** [comment]
Ah yeah I figuerd out why, I will fix it.
> I kinda always use the -f flag because the internal downloader seems buggy rn.
Yeah its just m3u8 parsing its annoying because of gogoanime who always change their stufff.

**Comment by sdaqo** [comment]
> @sdaqo OMG I know why this is happening... My "Bugfix" introduced this behaviour

It does? Now im confused I thought it was some other reason

**Comment by sdaqo** [comment]
but yeah seems like it

**Comment by sdaqo** [comment]
I changed it back again now it should work

**Comment by dymattic** [comment]
> Ah yeah I figuerd out why, I will fix it.
> 




> > @sdaqo OMG I know why this is happening... My "Bugfix" introduced this behaviour
> 
> It does? Now im confused I thought it was some other reason

Uhm I thought so at first.
But I think the problem lies in the download class, which should check wether the episode has been downloaded or not.

I thought my bugfix might have caused the new issue, because earlier the get_latest episodes method started at last episode+1

I am not sure myself though xD

**Comment by sdaqo** [comment]
@dymattic  Ah now I get it. Ok so here it goes: the episode that you put after adding a anime , is the latest episode you watched , so when you put 1 the script thinks, ahh thats the last ep he watched, the newest will be the next + everything after that. We have to change the prompt there so it doesnt get confusing and a option to put "0" there to starrt at the beginning

**Comment by sdaqo** [comment]
now it is fixed

**Comment by dymattic** [comment]
@sdaqo Omg yea this makes sense ... xD wow

**Pull request merged**